Adding example. (#2)
Cogmasters/concord-docs: docs/source/discord/resources/channel/methods/add_thread_member.rst

=================

.. doxygenfunction:: discord_add_thread_member

Example
-------

.. code:: c
   

>    // The channel that you executed the channel must be a thread.

Reviewer: ```suggestion
   // channel_id must be from a thread channel
```

>    discord_add_thread_member(client, msg->channel_id, msg->author.id); 

Reviewer: ```suggestion
   discord_add_thread_member(client, msg->channel_id, msg->author->id); 
```
